From 3776c4a41e268d7733e3476743d8104e48477c52 Mon Sep 17 00:00:00 2001
From: Adrien Nader <adrien.nader@canonical.com>
Date: Mon, 1 Jul 2024 16:49:06 +0200
Subject: [PATCH] configuration: read crypto-config and /etc/ssl/openssl.conf.d
 dropins
Forwarded: not-needed
Last-Update: 2024/07/01

Append two .include directives to the default configuration in order to
integrate with the crypto-config framework and to allow easier
configuration by sysadmins.

---
 apps/openssl.cnf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/apps/openssl.cnf b/apps/openssl.cnf
index 2833b6f30..175fc5897 100644
--- a/apps/openssl.cnf
+++ b/apps/openssl.cnf
@@ -388,3 +388,6 @@ oldcert = $insta::certout # insta.cert.pem
 # Certificate revocation
 cmd = rr
 oldcert = $insta::certout # insta.cert.pem
+
+.include /var/lib/crypto-config/profiles/current/openssl.conf.d
+.include /etc/ssl/openssl.conf.d
-- 
2.40.1

